### ISSUE TITLE
Fails to run without arguments

### ISSUE THREAD

**Issue by orhun** [dustinblackman/cargo-run-bin#20]
I recently integrated `cargo-run-bin` to my project: https://github.com/orhun/daktilo/commit/1addda6d3a4f7bd5f751815e9ebb493d2f3293a4

However, I'm getting an error when I run `cargo-run-bin` without arguments:

```
$ cargo bin

thread 'main' panicked at src/cli.rs:134:31:
index out of bounds: the len is 2 but the index is 2
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by dustinblackman** [comment]
Thanks for reporting!
Good catch! I'll fix that up.

**Comment by dustinblackman** [comment]
Fixed in 1.7.0. Thanks!

**Comment by orhun** [comment]
Just tested 1.7.0 and I can confirm the fix. Thanks!
